js/CRM_Custom_Form_Field.js

```diff
@@ -1,37 +1,35 @@
 CRM.$(function($) {
-  $(document).ready(function(){
```

Comment:
@lucky091588 Note for the future: `CRM.$(function($)` already does what `$(document).ready()` would do, so `$(document).ready()` is not needed.